site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-3.0.x-and-3.11.x-End-of-Life-Announcement.adoc

```diff
@@ -0,0 +1,28 @@
+= Farewell to Apache Cassandra 3.0.x and 3.11.x - End of Life Announcement
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: May 15, 2023
+:page-post-author: The Apache Cassandra Community
+:description: Apache Cassandra 3.0.x and 3.11.x - End of Life Announcement
+:keywords: 
+
+Dear Apache Cassandra Community,
+
+It is with mixed feelings that we announce the end of life (EOL) for Apache Cassandra versions 3.0 and 3.11, effective from the release of version 5.0 in Q4 of 2023. For more than half a decade, these versions have been a vital part of the open-source community, empowering countless users and organizations to build, maintain, and scale distributed databases easily and efficiently. Since the initial release of version 3.0 in November 2015, and the release of 3.11 in June 2017, we have seen enormous growth in the adoption of Apache Cassandra. With your support, these versions have successfully tackled challenges in data management, performance, and scalability. They have been the foundation for numerous applications and businesses around the globe, helping them thrive in today's data-driven world.
```

Comment:
How about adding "planned" in case it gets delayed? 🙂

```suggestion
It is with mixed feelings that we announce the end of life (EOL) for Apache Cassandra versions 3.0 and 3.11, effective from the [planned] release of version 5.0 in Q4 of 2023. For more than half a decade, these versions have been a vital part of the open-source community, empowering countless users and organizations to build, maintain, and scale distributed databases easily and efficiently. Since the initial release of version 3.0 in November 2015, and the release of 3.11 in June 2017, we have seen enormous growth in the adoption of Apache Cassandra. With your support, these versions have successfully tackled challenges in data management, performance, and scalability. They have been the foundation for numerous applications and businesses around the globe, helping them thrive in today's data-driven world.
```